lighthouse-core/gather/connections/connection.js

```diff
@@ -46,15 +46,16 @@ class Connection {
    * Call protocol methods
    * @param {!string} method
    * @param {!Object} params
+   * @param {Object=} options
    * @return {!Promise}
    */
-  sendCommand(method, params = {}) {
+  sendCommand(method, params = {}, options = {}) {
```

Comment:
sg, but can rename to `cmdOpts` or something? so it's clear this is different from our beloved `options` friend?